app/controllers/home_controller.rb

```diff
@@ -3,10 +3,10 @@ def about
   end
 
   def index
-    @posts = Post.published
+    @posts = Post.published.order(:published_at)
   end
 
-  def slug
+  def post
```

Comment:
To get the post's page. Again, it shouldn't be here, it is for development. It is going to get its own controller.